app/utils/__tests__/reducerInjectors.test.js

```diff
@@ -96,5 +98,11 @@ describe('reducer injectors', () => {
 
       expect(store.replaceReducer).toHaveBeenCalledTimes(2);
     });
+
+    it('should flush the persisted after injecting reducer', () => {
+      store.persistor.flush = jest.fn();
+      injectReducer('test', reducer);
+      expect(store.persistor.flush).toHaveBeenCalledTimes(1);
```

Comment:
Shouldn't you check if `flush()` is called before `injectReducers()`?

Reply:
I've added an additional test which checks the call order, and if flush is called before replaceReducer.